### DISCUSSION TITLE
Integration of Linkerd and NSM

### DISCUSSION THREAD

**Discussion by NikitaSkrynnik** [linkerd/linkerd2#10346] [Show and tell] [answered]
Hello, we've made an integration of Linkerd with our project [Network Service Mesh (NSM)](https://networkservicemesh.io/). We would like to show it or contribute it to somewhere. How can we do it?

The demo is still in progress but will be finished soon.
Link to the demo: https://github.com/NikitaSkrynnik/deployments-k8s/tree/linkerd/examples/interdomain/nsm_linkerd

**Comment by wmorgan** [comment, marked as answer]
@NikitaSkrynnik that's awesome. Would you be interested in contributing a blog post on linkerd.io? We can potentially syndicate to cncf.io as well.

**Reply by NikitaSkrynnik** [reply]
A blog post sounds good. Could you tell about a format of the post?

**Reply by wmorgan** [reply]
Sure, make a PR against linkerd/website that follows the format of e.g. https://github.com/linkerd/website/blob/main/linkerd.io/content/blog/2023/0130-mtls-and-linkerd.md . You'll have to pick a nice image as well.

**Reply by NikitaSkrynnik** [reply]
Thank you very much. We will prepare PR when we finish our integration
